web/src/components/CameraImage.jsx

```diff
@@ -45,12 +46,18 @@ export default function CameraImage({ camera, onload, searchParams = '', stretch
 
   return (
     <div className="relative w-full" ref={containerRef}>
-      <canvas data-testid="cameraimage-canvas" height={scaledHeight} ref={canvasRef} width={scaledWidth} />
-      {!hasLoaded ? (
-        <div className="absolute inset-0 flex justify-center" style={`height: ${scaledHeight}px`}>
-          <ActivityIndicator />
-        </div>
-      ) : null}
-    </div>
+      {
+        (enabled) ?
+          <canvas data-testid="cameraimage-canvas" height={scaledHeight} ref={canvasRef} width={scaledWidth} />
+          : <div class="text-center text-green-500">Camera is disabled in config, no stream or snapshot available!</div>
```

Comment:
I think leaving this as white text would be better.

Not 100% sure if this is how we will want this to look in the end (since we are also going to address the situation where a camera has an error to replace the "green screen") but it's probably good for now until it is addressed in a separate PR.